### PR TITLE
feat(consumer): Many namespace

### PR DESCRIPTION
**Pull request by DreamingOfDreams**
In annotation additional namespace, supports multiple different namespaces to add multiple different namespaces in the project.

### CODE REVIEW

**Comment by majusko** [comment]
Thank you. I will do a code review.

**Comment by majusko** [comment]
@DreamingOfDreams Thank you very much for your contribution to the project.  It can be merged right after applying some minor changes from the code review.

**Comment by DreamingOfDreams** [comment]
You're welcome. I hope the project is getting better and better. Please forgive my plastic English. I hope more and more people will use it and contribute to it.

**Comment by majusko** [comment]
@DreamingOfDreams Not in hurry but when are you planning to implement the required changes in a code review? I plan to do a patch release soon and I would like to include your changes there as well.

**Comment by DreamingOfDreams** [comment]
okay, no problem

**Review by majusko — CHANGES_REQUESTED** [review]

**Comment by majusko** [comment]
@DreamingOfDreams Sorry, I did not hit the "Submit change request" button. That is why I was waiting for a change and you did not see the request 🤦‍♂️

File: src/main/java/io/github/majusko/pulsar/consumer/ConsumerAggregator.java
Comment (DreamingOfDreams) [reply src/main/java/io/github/majusko/pulsar/consumer/ConsumerAggregator.java:57]: okay, no problem

**Comment by DreamingOfDreams** [comment]
@majusko OK, no problem. There are too many things in the company recently. It will be updated later for the Spring Festival in a few days. I'm sorry.

**Comment by shodo** [comment]
Hi, I need this feature too, when it will be merged? Can I help somehow?

**Comment by majusko** [comment]
Hey @shodo, There is a code review with changes that are required before merging. @DreamingOfDreams When do you think you can have a look into this?

**Comment by shodo** [comment]
If @DreamingOfDreams is busy I can try to take his branch and make the requested changes. I'm not familiar with GitHub workflow (I use bitbucket at work) but I can try

**Comment by majusko** [comment]
@shodo I'm afraid that @DreamingOfDreams forget to commit `UrlBuildService` file that is the most important change because that is the file where the namespace is hardcoded to the default one now. But I think you can try to make it happen based on his code. It shouldn't be that complicated change and you have some sort of base here.

**Comment by shodo** [comment]
@majusko Ok I'll give a try!

**Comment by shodo** [comment]
Hi @majusko I've finally opened a PR. Sorry for that delay but I had a lot of work these days. 
Any chance to merge it?

https://github.com/majusko/pulsar-java-spring-boot-starter/pull/227

**Comment by majusko** [comment]
Merged, fixed a few issues and released in `1.1.1`

**Comment by shodo** [comment]
@majusko Thanks a lot!

**Pull request merged**
